conversation_classification/kaggle/model.py

```diff
@@ -171,23 +243,16 @@ def main():
       tf.logging.info('Running in verbose mode')
       tf.logging.set_verbosity(tf.logging.DEBUG)
 
-    # Load and split data
-    tf.logging.debug('Loading data {}'.format(FLAGS.train_data))
-    data = WikiData(FLAGS.train_data)
-
-    x_train_text, x_test_text, y_train, y_test \
-      = data.split(TRAIN_PERCENT, FLAGS.y_class, DATA_SEED)
+    if os.path.isdir(FLAGS.model_dir):
```

Comment:
Could you use timestamps to version models or at least have overwriting pre-trained models be a flag? In general, holding on to previous versions of models will prove useful.

Reply:
I agree, although I'm explicitly saving the model below in a `saved_model` directory which adds a subdirectory with the timestamp of runtime for each model. So the `model` directory does get overwritten each time, but all the final models are saved separately. Which seems ok?

Reply:
Ah, ok. That should be fine.